pages/pricing.tsx

```diff
@@ -46,100 +36,107 @@ export default function PricingPage() {
               <Card
-                className="pricingCard pt-5 pb-4
-                 shadow-lg h-100"
-                border="primary"
-                bg="secondary"
-                text="white"
+                className="mx-auto col-12 p-3
+                 shadow-lg h-100 text-start"
+                bg="info"
+                style={{ minHeight: "468px" }}
               >
                 <Card.Body>
-                  <Card.Title>Standard Cloud</Card.Title>
+                  <div className="h3">Standard Cloud</div>
                   <h5>Starting at $150/month</h5>
-                  For companies who want to sync up to 100k Records to their
+                  For companies wanting to sync up to 1MM Records to their
                   destinations.
                   <br />
                   <br />
-                  <div style={boxTextStyle}>
+                  <div className={styles.pricingList}>
                     <b>Features include:</b>
-                    <li>Up to 100k Records synced/month</li>
-                    <li>Access to all Sources and Destinations</li>
+                    <li>Up to 1MM Records synced/month</li>
+                    <li>Tiered pricing based on monthly usage</li>
+                    <li> Access to all Sources and Destinations</li>
                     <li>
+                      {" "}
                       A no-code web UI allowing team members to access data
                     </li>
-                    <li>Business hours support</li>
+                    <li> Business hours support</li>
                   </div>
                 </Card.Body>
-                <Card.Footer style={{ borderTop: "none" }}>
-                  <Button variant="primary" href="/trial">
-                    Start free trial
-                  </Button>
-                </Card.Footer>
-                <small>No credit card required</small>
+                <Button
+                  variant="primary"
+                  className="w-100 mx-auto"
+                  href="/trial"
+                >
+                  Start free trial
+                </Button>
+                <small className="text-center pt-1">
+                  No credit card required
+                </small>
               </Card>
             </Col>
-            <Col className="mb-4 offset-md-3 offset-xl-0">
+
+            <Col className="mb-4 mx-auto ">
               <Card
-                className="pricingCard py-5 shadow-lg h-100"
-                border="primary"
-                bg="secondary"
-                text="white"
+                className="mx-auto p-3 shadow-lg h-100 col-12"
+                bg="white"
+                style={{ minHeight: "468px" }}
               >
-                <Card.Body>
-                  <Card.Title>Enterprise Cloud</Card.Title>
+                <Card.Body className="d-flex mx-auto align-items-start flex-column">
+                  <div className="h3">Enterprise Cloud</div>
```

Comment:
This div's text is centered, but the other cards' divs are left-aligned. Should probably be consistent?

Reply:
Ah ha!  Yep.